src/app/infrastructure/services/release.service.ts

```diff
@@ -0,0 +1,27 @@
+import { Injectable } from '@angular/core';
+
+@Injectable({
+    providedIn: 'root'
+})
+export class ReleaseService {
```

Comment:
I wonder what the purpose of is this services exactly?

Reply:
To make this service a singleton. When we declare on component. That make sure this service is a singleton.